tictactoe/implementation/src/main/kotlin/com/anaplan/engineering/azuki/tictactoe/implementation/GameManager.kt

```diff
@@ -36,16 +39,28 @@ class GameManager(private val store: File) {
             throw IllegalArgumentException("Game '$name' already exists")
         }
         games[name] = game
-        Log.info("Added game:\n$game")
+        Log.info("Added game: (${game::class.simpleName})\n$game")
         Log.info("Active games = ${games.keys}")
         return game
     }
 
     private val objectMapper = ObjectMapper()
         .registerModule(KotlinModule())
 
+    val gameCreator: GameCreator by lazy {
+        val loader = ServiceLoader.load(GameCreator::class.java)
+        loader.iterator().asSequence().first()
```

Comment:
⛏️ Given that `first` also throws an exception when the sequence is empty, what's the benefit of this over
`loader.iterator().next()`?

Reply:
It had simply been copied from another load where we  expected >=1